client/src/components/Board/Column.tsx

```diff
@@ -1,19 +1,28 @@
-import { Grid, Typography } from '@material-ui/core';
+import { useState } from 'react';
+import { Grid, Button } from '@material-ui/core';
 import useStyles from './useStyles';
 import MoreHorizIcon from '@material-ui/icons/MoreHoriz';
 import { Draggable, Droppable } from 'react-beautiful-dnd';
-import { IPropColumn, IPropTask } from '../../interface/Board';
+import { IPropColumn, IPropTask, INewTask } from '../../interface/Board';
 import Task from './Task';
+import NewTaskForm from './Forms/NewTaskForm';
 import { theme } from '../../themes/theme';
 
 interface Props {
   column: IPropColumn;
   tasks: IPropTask[];
   index: number;
+  addTask: (newTask: INewTask) => void;
 }
 
-export default function Column({ column, tasks, index }: Props): JSX.Element {
+export default function Column({ column, tasks, index, addTask }: Props): JSX.Element {
   const classes = useStyles(theme);
+  const [showNewTask, setShowNewTask] = useState(false);
+
+  const handleClick = () => {
+    setShowNewTask(!showNewTask);
```

Comment:
You can use call back form e.g setOpen((prevOpen) => !prevOpen); 
https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous